test/cell_test.rb

```diff
@@ -2,7 +2,12 @@
 require_relative "../lib/cell.rb"
 
 describe Cell do
+  let(:cell) {Cell.new("B4")}
   it "should exist" do
-    expect(Cell.new("B4")).to be_kind_of(Cell)
```

Comment:
This test is not using the  cell symbol instead its using the whole class if you may please change this.